pythoncz_tests/models/github_test.py

```diff
@@ -0,0 +1,432 @@
+import re
+import warnings
+
+import pytest
+import requests
+import responses
+from werkzeug.contrib.cache import NullCache
+
+from pythoncz.models import github as github_module
+from pythoncz_tests.models import github_test_fixtures as fixtures
```

Comment:
I didn't find out how to pass arguments to pytest fixtures. I had them, then realized I'm probably overcomplicating things and then I decided to make them just simple functions, taking arguments and producing fixtures. If you know about a way how to make the tests cleaner and if turning the module into pytest fixtures contributes to that goal, go for it 👍 But I suggest that as a next step to this PR.